packages/components/src/inputs/SelectInput/SelectInput.tsx

```diff
@@ -35,6 +36,7 @@ export function SelectInput(props: SelectInputProps) {
     options,
     disabled,
     disableVariant = 'default',
+    outlined = true,
```

Comment:
Why not to use here Mui Select **variant** prop? (it should be variant = "standard")